src/css/style.css

```diff
@@ -0,0 +1,535 @@
+/*
```

Comment:
If you only import the base from tailwind using:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

you shouldn't have to copy all this from Tailwind